### DISCUSSION TITLE
How do we pick a selection of fields from a collection?

### DISCUSSION THREAD

**Discussion by Innominus** [pocketbase/pocketbase#1654] [Q&A]
Currently using Pocketbase without the Dart/Js library. 
How do I query a Pocketbase collection and only get back a specific field?
Say I have a collection of recipes, how do I just call the collection and only receive back the id column/field from this collection?
Thanks :)

**Comment by ganigeorgiev** [comment]
This is not supported at the moment (unless you define a [custom endpoint](https://pocketbase.io/docs/router/#register-new-route)).

_You can also check the other similar discussions - https://github.com/pocketbase/pocketbase/discussions/1648, https://github.com/pocketbase/pocketbase/discussions/1605, https://github.com/pocketbase/pocketbase/discussions/354._

**Reply by Innominus** [reply]
Hey, that's my bad. I think I used poor search terms. Thanks for that. 
I think in the meantime while that's getting added I'll probably setup a separate table that contains all the IDs or something similar.
